modules/apps_api/app/swagger/apps_api/v0/apps.yml

```diff
@@ -0,0 +1,123 @@
+openapi: '3.0.0'
+info:
+  version: 0.0.1
+  title: Apps
+  description: |
+    The Applications Api provides a list of applications
+
+    ## Design
+
+    ### Authorization and Access
+
+    This API is available publically without an API Key.
+
+    1. Client Request: GET https://sandbox-api.va.gov/services/apps/v0/directory
+
+    2. Service Response: A JSON API object with a list of all applications available
+
+
+  termsOfService: 'https://developer.va.gov/terms-of-service'
+  contact:
+    name: Vets.gov
```

Comment:
This should be va.gov not vets.gov. We probably havent updated it in other modules yet.